x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/create_rules_route.test.ts

```diff
@@ -29,8 +32,12 @@ describe('create_rules', () => {
   let clients = clientsServiceMock.createClients();
 
   beforeEach(() => {
+    // jest carries state between mocked implementations when using
+    // spyOn. So now we're doing all three of these.
+    // https://github.com/facebook/jest/issues/7136#issuecomment-565976599
```

Comment:
Thanks for the link and the interesting comment that talks about it. I think that maybe our tests and techniques are growing to such a degree we almost need a bit more stable location for all of our test utils and workaround otherwise I would maybe move this section to a localized `__mocks__/utils.ts` to reduce boilerplate?

Optional: this is still fine as is.